src/Razor/test/Microsoft.AspNetCore.Razor.Language.Test/TestFiles/IntegrationTests/ComponentRuntimeCodeGenerationTest/ComponentImports/_Imports.codegen.cs

```diff
@@ -16,6 +16,7 @@ public class _Imports : System.Object
         #pragma warning disable 1998
         protected void Execute()
         {
+            builder.AddContent(0, Foo);
```

Comment:
One of these days we should nuke all of the content from the _Imports runtime output file >_<